.eslintrc.js

```diff
@@ -125,6 +126,7 @@ module.exports = {
     'local/await-expect': 2,
     'local/closure-type-primitives': 2,
     'local/dict-string-keys': 2,
+    'local/forbidden-terms': [2, getForbiddenTerms],
```

Comment:
I previously wrote this to also check missing terms, but that's only useful for the license header. Since we're already using **`notice/notice`** for this purpose, I thought it would be simpler to only check forbidden terms.

https://github.com/ampproject/amphtml/blob/ae162dff45078e87ce6da09b475f1d8c63216c20/build-system/tasks/presubmit-checks.js#L1256-L1261

Reply:
I've always been thankful that this check existed. When you added `notice/notice`, the experience of creating a new JS file, hitting save, and having the copyright auto-appear was delightful. This PR is going several steps further with forbidden terms. Nice work!